python_modules/dagster/dagster/_daemon/auto_materialize_daemon.py

```diff
@@ -0,0 +1,100 @@
+import dagster._check as check
+from dagster._core.definitions.asset_reconciliation_sensor import (
+    AssetReconciliationCursor,
+    reconcile,
+)
+from dagster._core.definitions.external_asset_graph import ExternalAssetGraph
+from dagster._core.definitions.selector import PipelineSelector
+from dagster._core.storage.pipeline_run import DagsterRunStatus
+from dagster._core.workspace.context import IWorkspaceProcessContext
+from dagster._daemon.daemon import DaemonIterator, IntervalDaemon
+
+CURSOR_NAME = "ASSET_RECONCILIATION_DAEMON_CURSOR"
```

Comment:
I think we should try to avoid "reconciliation" in this name unless we're using it elsewhere.